src/main/java/com/nextstep/camp/lotto/domain/entity/LottoTickets.java

```diff
@@ -0,0 +1,46 @@
+package com.nextstep.camp.lotto.domain.entity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.nextstep.camp.lotto.domain.exception.LottoTicketsCannotBeEmptyException;
+import com.nextstep.camp.lotto.domain.strategy.LottoPickStrategy;
+import com.nextstep.camp.lotto.domain.type.MatchResult;
+import com.nextstep.camp.lotto.domain.vo.WinningNumbers;
+
+public class LottoTickets {
+    private final List<LottoTicket> tickets;
+
+    private LottoTickets(LottoPickStrategy pickStrategy) {
+        List<LottoTicket> tickets = pickStrategy.pick();
+        if (tickets == null || tickets.isEmpty()) {
+            throw new LottoTicketsCannotBeEmptyException();
+        }
+        this.tickets = tickets;
+    }
```

Comment:
생성자 메서드는 객체 생성을 위한 유효성 검사와 초기화에 집중하면 어떨까요?
```suggestion
    private LottoTickets(List<LottoTicket> tickets) {
        // validate
        this.tickets = tickets;
    }
```